server/metrics.go

```diff
@@ -46,6 +47,15 @@ func NewServerMetrics(cfg Config) *Metrics {
 			NativeHistogramMaxBucketNumber:  100,
 			NativeHistogramMinResetDuration: time.Hour,
 		}, []string{"method", "route", "status_code", "ws"}),
+		PerTenantRequestDuration: reg.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace:                       cfg.MetricsNamespace,
+			Name:                            "per_tenant_request_duration_seconds",
+			Help:                            "Time (in seconds) spent serving HTTP requests for a particular tenant.",
+			Buckets:                         instrument.DefBuckets,
```

Comment:
Hi, do you really need the classic histogram series? I know that currently exemplars don't work without the classic buckets, but maybe you don't need exemplars per tenant anyway and it will be solved as well

Reply:
I was planning to subtract a particular tenant from our existing SLO metrics, which rely on classic histograms (AFAIK) with the same buckets. So I do think I rely on them (until the main SLOs) use native histograms.